### ISSUE TITLE
Error - Required 'compute.instances.list'

### ISSUE THREAD

**Issue by BlenderIsaac** [kashifulhaque/discord-bot-for-gcp#6]
I tried to set up this bot on replit.com so it would be possible to start up a server on GCP.
Once I had put the necessary information in when i tried to run it it would not work because it gave an error.

ApiError: Required 'compute.instances.list' permission for 'PROJECTNAME'

Is there any way around this issue? I assume I need to give the service account the permission 'compute.instances.list' but I don't know how.

**Comment by kashifulhaque** [comment]
Yeah, I do think you might need to give service account access for that particular project, but the problem with GCP docs is that it's quite vast and even I get lost while looking at it for times.
I hope you find the thing you're looking for on GCP docs
